readthedocs/templates/profiles/private/delete_account.html

```diff
@@ -9,6 +9,44 @@
 {% block edit_content_header %} {% trans "Delete Account" %} {% endblock %}
 
 {% block edit_content %}
+  {% if projects_to_be_deleted %}
```

Comment:
If this is a user with an organization, then this query will return `[]`. We should probably account for that here, but looping over all the Organizations' projects, and noting that they will also be deleted.

Reply:
Does it make sense to show those as a nested list when listing the organizations?

Reply:
Yea, I think a nested list is correct. Annoying the styles for that don't just work :(